api/generate.go

```diff
@@ -32,7 +32,7 @@ func Generate(cfg *config.Config, option ...Option) error {
 			for _, v := range cfg.Sources {
 				cfg.Federation.Version = 1
 				urlString := urlRegex.FindStringSubmatch(v.Input)
-				if urlString != nil && urlString[1] == "https://specs.apollo.dev/federation/v2.0" {
+				if urlString != nil && urlString[1] == "https://specs.apollo.dev/federation/v2.7" {
```

Comment:
I think this probably should stay at `2.0` or maybe should just check if federation version starts with 2?

Reply:
[b33376a](https://github.com/99designs/gqlgen/pull/2876/commits/b33376a71e320af6c2fb455bd631d3ff9d152585)